minet/instagram/formatters.py

```diff
@@ -91,14 +110,44 @@ def format_user_post(item):
 
     text = getpath(item, ["caption", "text"])
 
-    hashtags = None
-    mentioned_names = None
+    hashtags = []
+    mentioned_names = []
 
     if text:
-        hashtags = "|".join(extract_hashtags(text))
-        mentioned_names = "|".join(extract_handles(text))
+        hashtags = extract_hashtags(text)
+        mentioned_names = extract_handles(text)
+
+    medias_url = []
+    medias_type = []
+    usertags_medias = []
```

Comment:
Use a set to handle duplication (this is linked to medialab/casanova#61) but can be dealt with by casting the set as a list when giving it to the namedrecord later on if casanova is not fixed earlier.